pkg/plugins/resources/yaml/main.go

```diff
@@ -8,23 +8,23 @@ import (
 
 	"github.com/mitchellh/mapstructure"
 	"github.com/sirupsen/logrus"
+	"github.com/updatecli/updatecli/pkg/core/result"
 	"github.com/updatecli/updatecli/pkg/core/text"
 
 	"gopkg.in/yaml.v3"
 )
 
-var (
-	yamlIdent int = 2
-)
-
 // Spec defines a specification for a "yaml" resource
 // parsed from an updatecli manifest file
 type Spec struct {
-	// Specifies the YAML file
+	// [source,condition,target] File contains the file path to take in account
 	File string `yaml:",omitempty"`
-	// Specifies the YAML key
+	// TODO: implement files for condition too
```

Comment:
```suggestion
```
I prefer to implement multiple files for conditions too in another pull request, as I don't see the immediate use case(s).